modules/rules_apple_linker/0.3.0/presubmit.yml

```
@@ -0,0 +1,10 @@
+tasks:
+  verify_targets:
+    name: Verify targets
+    platform: macos
+    build_targets:
+      - '@rules_apple_linker//:lld'
+      - '@rules_apple_linker//:zld'
+      - //test/...
+    test_targets:
+      - //test/...
```

```suggestion
      - `@rules_apple_linker//test/...`
```

This is because the tests are run from an anonymous root module, so `//test/...` would end up in that one.